### ISSUE TITLE
New Battery For Kitchen Handheld Vacuum Cleaner? 🔋 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#302]
@iteles you have noted on several occasions that the kitchen handheld vac 
doesn't charge for you and when it does the battery life is poor. 🪫 

This is an easy fix without having to buy a new vac unit; just replace the battery:

https://www.amazon.es/dp/B07BFSKFN1/ref=asc_df_B07BFSKFN1
<img width="1310" alt="image" src="https://github.com/user-attachments/assets/ef283045-c4e1-4461-b741-0cb6fa86b6f0">

# Todo

+ [ ] @iteles to confirm if the vac is working (it works for me) 🙏 
+ [ ] if not, decide if a new battery will solve the problem - even if we have to charge it externally using a charger and the attach it to the vac when in-use. 💭

**Comment by iteles** [comment]
Do we not have any spare Bosch batteries we could use to test this? I don't know if the battery is the problem or if it's the charger or if it's just the vacuum cleaner that doesn't charge and therefore a battery won't help 🤷🏻‍♀️

**Comment by nelsonic** [comment]
No. The only batteries we have are much bigger and therefore don’t fit in the vac charger (wall mount/dock).

**Comment by iteles** [comment]
Then yea, when you check out of Amazon, please include the battery.
We can just be careful with it and if it doesn't work and the problem is not the battery, we can return it.

**Comment by nelsonic** [comment]
The battery is in the shopping cart but I don’t need anything else right now so if you do, please include it in *your* checkout. 🛒👌

**Comment by iteles** [comment]
That ship sailed. I suggested you include it in your checkout because I didn't need anything. 

But I'm placing an order from Amazon U.K. soon and can order from there more cheaply anyway: 

https://amzn.eu/d/5ItUEcM

![image](https://github.com/user-attachments/assets/3160635d-b423-47c3-b822-05d9f716446e)

@nelsonic Can you please check this is correct so I can go ahead and order?

**Comment by nelsonic** [comment]
@iteles "cheaper" but _**Compatible** not **Original**_; so no thank you.  👎 
Black Friday is next week, it will be cheaper again. ⏳
